modules/installation-nutanix-ccm.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc
+
+:_content-type: PROCEDURE
+[id="nutanix-ccm-config_{context}"]
+= Adding the ConfigMaps and Secret needed for Nutanix CCM
```

Comment:
```suggestion
= Adding the ConfigMaps and Secret required for Nutanix CCM
```

Reply:
Same here too re: either going with general reference lower case (probably preferred in this case since we can't use backticks in headings):

> Adding the config maps and secret required for Nutanix CCM

Or object references:

> Adding the ConfigMaps and Secret objects required for Nutanix CCM

Another idea, possibly a slight rewording to make it a little shorter, though don't know if it's necessarily better:

> Adding required configs and secrets for Nutanix CCM